### PR TITLE
[WIP][1LP] test_multiple_host_bad_creds: update RHV provider error message

### PR DESCRIPTION
**Pull request by istein1**
__Fixing__ 
test_multiple_host_bad_creds expected error message, or RHV provider because it is currently expecting error that is correct for VMWare, but not for RHV provider.

### CODE REVIEW

**Comment by istein1** [comment]
Verified on RHV provider.

File: cfme/tests/infrastructure/test_host.py [hunk 1]
```diff
@@ -87,6 +92,6 @@ def test_multiple_host_bad_creds(setup_provider, provider):
                            'validate_host': cfme_host["name"]})
 
     sel.click(credential_form.validate_multi_host)
-    flash.assert_message_match('Cannot complete login due to an incorrect user name or password.')
+    flash.assert_message_match(msgs[provider.type])
```
Comment (izapolsk) [inline cfme/tests/infrastructure/test_host.py]: REQUIRED: @istein1, we are trying to get away from using provider.type and et cetra. 
Please replace this with provider.one_of() construction.

Comment (RonnyPfannschmidt) [inline cfme/tests/infrastructure/test_host.py]: @izapolsk we dont have a way to do a lookup by general provider category atm - so this use is valid and one-of is not usefull

Comment (izapolsk) [inline cfme/tests/infrastructure/test_host.py]: @RonnyPfannschmidt, I'm not sure I got you. 
msgs dict has definitions only for two provider types.

so, this statement can be easily replaced with 
```
if provider.one_of(RHEVMProvider):
 msg = 'blabla'
elif provider.one_of(VMwareProvider):
 msg = 'blabla again'
```
btw, what is the reason for having only two provider types in that dict ? What about other providers ?

**Comment by istein1** [comment]
This PR is not relevant any more. It was replaced by another PR:
https://github.com/ManageIQ/integration_tests/pull/4895

**Pull request closed without merging**